boards/arm/tmo_dev_edge/tmo_dev_edge_defconfig

```diff
@@ -13,6 +13,20 @@ CONFIG_SERIAL=y
 CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=38400000
 CONFIG_GPIO=y
+CONFIG_POSIX_CLOCK=y
+CONFIG_CMU_HFCLK_HFXO=y
+CONFIG_DEVICE_SHELL=y
+CONFIG_GPIO_SHELL=y
+CONFIG_I2C=y
+CONFIG_SPI=y
+CONFIG_REBOOT=y
+CONFIG_COUNTER=y
+CONFIG_LED=y
+CONFIG_LED_PWM=y
+CONFIG_LED_SHELL=y
```

Comment:
Do we actually need all of this in the defconfig?

Reply:
For the shell_module, but once we gat onto tmo_shell again, we can revert to using it.